pyat/at/lattice/idtable_element.py

```diff
@@ -120,37 +49,156 @@ def set_IdTablePass(self):
     def get_PassMethod(self):
         return getattr(self, 'PassMethod')
```

Comment:
Dear @lfarv , at the moment of testing the Insertion Device I tipically needed to set the pass method to IdTablePass and DriftPass. I implemented these functions to profit from autocompletion therefore avoiding misspelling errors.

In any case, you are right, get_PassMethod is redudant. I have put a UserWarning about its deprecation. It might be possible to completely remove it in a following version. Or if you see it is better I would remove it now.

Reply:
Ok, I don' care, it's just unusual but it is harmless!